pkg/codegen/internal/test/testdata/simple-enum-schema/python/pulumi_plant_provider/_enums.py

```diff
@@ -0,0 +1,25 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+from enum import Enum
+
+__all__ = ["ContainerColor", "ContainerSize"]
```

Comment:
Should this match the similar codegen of `__all__` for `_inputs.py` (with each on its own line?)